templates/verify.html

```diff
@@ -1,118 +1,64 @@
 <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
-<html>
+<html xmlns="http://www.w3.org/1999/xhtml">
   <head>
     <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
-    <title>{{t "Firefox Accounts"}}</title>
-    <style>
-      a {
-        color: #0095dd;
-        text-decoration: none !important;
-      }
+    <title>{{t "Firefox Account Activation"}}</title>
+  </head>
 
-      @media only screen and (max-width: 599px){
-        /*CLIENT-SPECIFIC MOBILE STYLES */
-        body, table, td, p, a, li, blockquote{-webkit-text-size-adjust:none !important;} /* Prevent Webkit platforms from changing default text sizes */
-        body{width:100% !important; min-width:100% !important;} /* Prevent iOS Mail from adding padding to the body */
-        #bodyCell{padding:10px !important;}
+  <body style="-ms-text-size-adjust: 100%; -webkit-text-size-adjust: 100%; margin: 0; padding: 0;">
+    <table border="0" cellpadding="0" cellspacing="0" height="100%" width="100%" id="body-table" style="-webkit-text-size-adjust: 100%; border-collapse: collapse; mso-table-lspace: 0pt; mso-table-rspace: 0pt;">
+      <tr>
+        <td align="center" valign="top">
+          <table align="center" border="0" cellpadding="0" cellspacing="0" width="310" id="email-container" style="-webkit-text-size-adjust: 100%; border-collapse: collapse; mso-table-lspace: 0pt; mso-table-rspace: 0pt; width: 310px; margin: 0 auto;">
+            
+            <!--Logo-->
+            <tr>
+              <td align="center" id="firefox-logo" style="padding: 20px 0;">
+                <img src="http://image.e.mozilla.org/lib/fe9915707361037e75/m/2/fxlogojg.gif" height="95" width="88" alt="Firefox" style="-ms-interpolation-mode: bicubic;">
```

Comment:
Same as above, since we're saying XHTML 1.0, does this need to be `<img />`?
I'll see if I can run this through some HTML linter.